vickyctl/src/http_client.rs

```diff
@@ -0,0 +1,15 @@
+use crate::AppContext;
+use reqwest::blocking::Client;
+use reqwest::header::{HeaderMap, AUTHORIZATION};
+use crate::error::Error;
+
+pub fn prepare_client(ctx: &AppContext) -> Result<Client, Error> {
+    let mut default_headers = HeaderMap::new();
+    default_headers.insert(AUTHORIZATION, ctx.vicky_token.parse().unwrap());
+    let client = Client::builder()
+        .default_headers(default_headers)
+        // TODO?: .https_only(true)
```

Comment:
I would think, that this is not needed, since it would be very wierd behaviour downgrading an `https://` url...

Reply:
Who was talking about downgrading https? The line would only allow https traffic

Reply:
If you enter a http url as API url, like http://vicky.lab.wobcom.de, I would expect it to use http as given.
If you enter a https url, it will use HTTPS and won't downgrade at any time.

Therefore, I see no need for enforcing HTTPS here.

Reply:
i'm still confused why you think it'd downgrade anything

Reply:
Its not about you downgrading anything but not needing a https enforcement to avoid downgrading :)

Reply:
sorry if im autistic, but wouldn't going from http to https be upgrading?

Reply:
My initial thought was just to do this for safety reasons, but not doing it is fine with me too.